django-stubs/core/cache/backends/base.pyi

```diff
@@ -18,22 +18,22 @@ class BaseCache:
     key_func: Callable = ...
     def __init__(self, params: Dict[str, Any]) -> None: ...
     def get_backend_timeout(self, timeout: Any = ...) -> Optional[float]: ...
-    def make_key(self, key: Any, version: Optional[Any] = ...) -> str: ...
-    def add(self, key: Any, value: Any, timeout: Any = ..., version: Optional[Any] = ...) -> bool: ...
-    def get(self, key: Any, default: Optional[Any] = ..., version: Optional[Any] = ...) -> Any: ...
-    def set(self, key: Any, value: Any, timeout: Any = ..., version: Optional[Any] = ...) -> None: ...
-    def touch(self, key: Any, timeout: Any = ..., version: Optional[Any] = ...) -> bool: ...
-    def delete(self, key: Any, version: Optional[Any] = ...) -> None: ...
+    def make_key(self, key: Any, version: Optional[int] = ...) -> str: ...
+    def add(self, key: Any, value: Any, timeout: Any = ..., version: Optional[int] = ...) -> bool: ...
+    def get(self, key: Any, default: Optional[Any] = ..., version: Optional[int] = ...) -> Any: ...
+    def set(self, key: Any, value: Any, timeout: Any = ..., version: Optional[int] = ...) -> None: ...
+    def touch(self, key: Any, timeout: Any = ..., version: Optional[int] = ...) -> bool: ...
```

Comment:
Is `timeout` really `Any`?

Reply:
Nope, looking closer now, I see that they resolve the default arg that is set as `object()`:

https://github.com/django/django/blob/2a2bde52f31e09e95ce616e8e6bc0ffeb68f76c9/django/core/cache/backends/base.py#L85-L95

Looks like it should be an `int`. Let me change that.